core/src/baseView.js

```diff
@@ -108,7 +108,7 @@ class BaseView {
 
   id(name) {
     if (!this.idSet[name]) {
-      window.__ID++;
+      window.__ID = (typeof Android.getNewID == "function")? Android.getNewID() : (window.__ID + 1);
```

Comment:
Throwing error in non Android platforms.
"Uncaught ReferenceError: Android is not defined"